app/models/manageiq/providers/hawkular/middleware_manager/live_metrics_capture.rb

```diff
@@ -1,13 +1,9 @@
 module ManageIQ::Providers
   class Hawkular::MiddlewareManager::LiveMetricsCapture
-    class TargetValidationError < RuntimeError; end
     class MetricValidationError < RuntimeError; end
 
     def initialize(target)
       @target = target
-      unless @target.kind_of?(MiddlewareServer)
```

Comment:
Yes, now LiveMetricsCapture can be reused easily to cover other middleware models in a similar way as it does for datasources now.